python/ray/_private/ray_constants.py

```diff
@@ -437,6 +437,9 @@ def gcs_actor_scheduling_enabled():
 
 DEFAULT_RESOURCES = {"CPU", "GPU", "memory", "object_store_memory"}
 
+# Sync with src/ray/common/constants.h
```

Comment:
I remember it was possible to share the same variable between this & constant.h? Can we just do it?